azure-samples/src/main/java/com/azure/management/appservice/samples/ManageLinuxWebAppStorageAccountConnection.java

```diff
@@ -0,0 +1,199 @@
+
+            return true;
+        } catch (Exception e) {
+            System.err.println(e.getMessage());
+            e.printStackTrace();
+        } finally {
+            try {
+                System.out.println("Deleting Resource Group: " + rgName);
+                azure.resourceGroups().beginDeleteByName(rgName);
+                System.out.println("Deleted Resource Group: " + rgName);
+            } catch (NullPointerException npe) {
+                System.out.println("Did not create any resources in Azure. No clean up is necessary");
+            } catch (Exception g) {
+                g.printStackTrace();
+            }
+        }
+        return false;
+    }
+    /**
+     * Main entry point.
+     * @param args the parameters
+     */
+    public static void main(String[] args) {
+
+        try {
+
+            //=============================================================
+            // Authenticate
+
+            final File credFile = new File(System.getenv("AZURE_AUTH_LOCATION"));
+
+            Azure azure = Azure.configure()
+                    .withLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BASIC))
+                    .authenticate(credFile)
+                    .withDefaultSubscription();
+
+            // Print selected subscription
+            System.out.println("Selected subscription: " + azure.subscriptionId());
+
+            runSample(azure);
+        } catch (Exception e) {
+            System.out.println(e.getMessage());
+            e.printStackTrace();
+        }
+    }
+
+    private static BlobContainerClient setUpStorageAccount(String connectionString, String containerName) {
+        BlobContainerClient blobContainerClient = new BlobContainerClientBuilder()
+                .connectionString(connectionString)
+                .containerName(containerName)
+                .buildClient();
+
+        blobContainerClient.create();
+
+        BlobSignedIdentifier identifier = new BlobSignedIdentifier()
+                .setId("webapp")
+                .setAccessPolicy(new BlobAccessPolicy()
+                        .setStartsOn(OffsetDateTime.now())
+                        .setExpiresOn(OffsetDateTime.now().plusDays(7))
+                        .setPermissions("rl"));
+
+        blobContainerClient.setAccessPolicy(PublicAccessType.CONTAINER, Collections.singletonList(identifier));
+
+        return blobContainerClient;
+    }
+
+    private static void uploadFileToContainer(BlobContainerClient blobContainerClient, String fileName, String filePath) {
+        BlobClient blobClient = blobContainerClient.getBlobClient(fileName);
+        File file = new File(fileName);
+        try (InputStream is = new FileInputStream(file)) {
+            blobClient.upload(is, file.length());
+        } catch (IOException e) {
+            System.out.println(e.getMessage());
+            e.printStackTrace();
+        }
+    }
+}
```

Comment:
@xccc-msft this is migrate to latest storage blob client. might be helpful if you need migrate in compute.